distribution/docs/src/main/resources/_contents/_endpoints/cometd-endpoint-contents.adoc

```diff
@@ -0,0 +1,855 @@
+|String
+|Any
+|"remove" (Currently only used action)
+|If a client publishes with the `remove` action, it removes the notification with the given id from the persistence store.
+
+|id
+|ID of the notification to which the action relates
+|String
+|Any
+|"27ec3222af1144ff827a351b1962a236"
+|This is the id of the notification
+
+|===
+
+.Example: Notification Operation Request
+[source,json,linenums]
+----
+"data": [ {
+	"action": "remove",
+ 	"id": "27ec3222af1144ff827a351b1962a236"
+} ]
+----
+
+====== Persistence of Notifications
+
+Notifications are persisted between sessions, however due to the nature of CometD communications, they will not be visible at first connection/subscription to ``/${ddf-branding-lowercase}/notifications/**``.
+
+In order to retrieve notifications that were persisted or may have occurred since the previous session a client simply must publish an empty json message, `{}` to `/${ddf-branding-lowercase}/notifications`.
+This will return all existing notifications to the user.
+
+====== Activity Events Channel
+
+To receive all activity updates, follow the instructions at <<asynchronous_search_subscribing, Subscribing to Notifications>> and subscribe to `/ddf/activities/\*\*`
+
+Activity update messages follow a specific format when they are published to the activities channel.
+These messages contain a data map that encapsulates the activity information.
+
+.CometD Activity Format
+[cols="1m,3,1,1,1" options="header"]
+|===
+|Map Key
+|Description
+|Value Type
+|Possible Values
+|Example Values
+
+|category
+|Category of the activity
+|String
+|Any String
+|"Product Retrieval"
+
+|id
+|ID that uniquely identifies the activity that sent out the update. Not required
+to be unique per update.
+|String
+|Any String
+|"b72ccdf6-8ca7-4f53-a0f6-b0ad264393b0"
+
+|message
+|User-readable message that explains the current activity status
+|String
+|Any String
+|"Resource retrieval downloading."
+
+|operations
+|Map of operations that can be performed on this activity
+|JSON Map
+|A map of keys with populated values (that evaluate to 'true' rather than 'null' 'undefined' or 'false') These operations and their values can be used by clients to communicate back to the server by sending a message back on the same channel.
+
+If the value is a URL, the client should invoke the URL as a result of the user invoking the activity operation.
+
+a|
+[source,json,linenums]
+----
+"operations" : {
+	"download" : "http://example.com/product"
+}
+----
+
+If the value is not a URL, the client should send a message back to the server on the same topic with the operation name.
+
+Note: the ${ddf-branding} UI will interpret several values with special icons:
+* `download`
+* `retry`
+* `cancel`
+* `pause`
+* `remove`
+* `resume`
+
+|progress
+|Percentage value of activity completion
+|String
+|Integer between 0 - 100 followed by a %
+|"45%"
+
+|status
+|Enumerated value that displays the current state of the activity
+|String
+a|
```

Comment:
No. Asciidoctor seems to have trouble with certain formatting inside tables. Might be another argument for avoiding tables. Kind of cheated to at least suggest the list.